scripts/__init__.py

```diff
@@ -1,12 +1,12 @@
 import sys
+from pathlib import Path
 
 from loguru import logger
-from pathlib import Path
 
-pkg_path = [x for x in sys.path if x.endswith('site-packages')]
+pkg_path = [x for x in sys.path if x.endswith("site-packages")]
 
-if not Path(fr'{pkg_path[0]}/my_paths.pth').exists():
-    with open(fr'{pkg_path[0]}/my_paths.pth', 'w') as file:
+if not Path(rf"{pkg_path[0]}/my_paths.pth").exists():
+    with open(rf"{pkg_path[0]}/my_paths.pth", "w") as file:
         file.write(str(Path.cwd()))
```

Comment:
Why do you create this file? For debugging? It is used nowhere except in this file:

```sh
$ grep -R my_paths .
./scripts/__init__.py:if not Path(fr'{pkg_path[0]}/my_paths.pth').exists():
./scripts/__init__.py:    with open(fr'{pkg_path[0]}/my_paths.pth', 'w') as file:
```

Reply:
just a temporary workaround to mypy and pyright works properly. There's already an open issue to remove this.

Reply:
Would be good to add a small comment which bug this workaround addresses. Looks a bit weird to create a file in the first site-package.

Reply:
The issue is that `mypy` requires a `-stubs` package to be installed.  It won't look at a `-stubs` package anywhere else.  So the file is needed to tell `mypy` to look in the project directory, but when doing the test, we should create it, then remove it before/after running `mypy`

`pyright` does not have this issue.

Reply:
@twoertwein thanks for the feedback, you are absolutely right, probably a comment would clarify all the situation. @Dr-Irv  thx the idea <3